src/ui/main.qml

```diff
@@ -348,6 +348,82 @@ Window {
         }
     }
 
+    Connections {
+        target: VPNErrorHandler
+        function onSubscriptionGeneric() {
+            mainStackView.push("qrc:/ui/views/ViewErrorFullScreen.qml", {
+                // Problem confirming subscription...
+                headlineText: VPNl18n.GenericPurchaseErrorGenericPurchaseErrorHeader,
+
+                // An unexpected error has occurred, please try again.
+                // Visit our help center below to learn how to troubleshoot this issue or contact support.
```

Comment:
Are we missing a link or button to the 'help center below' ?

Reply:
Yea you are right! I did not think of that. 
One thing is that we always have the get help menu on the top therefore referencing "below" does not make sense.
I think Chris needs to look into this again because in the Figma spec we have this even when the "Sign Out" button is below.